<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Book</class>
 <widget class="QMainWindow" name="Book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Welcome to our Airlines</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Will you flying:</string>
          </property>
          <widget class="QRadioButton" name="local_radioButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>69</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Locally</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="international_radioButton">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>50</y>
             <width>91</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>Internationally</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="submit_pushButton">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Please fill your flight information</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Please select Your source:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="source_comboBox">
            <item>
             <property name="text">
              <string>Addis Ababa</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gonder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bahirdar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jimma</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adama</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Please select Your Destination:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="destination_comboBox">
            <item>
             <property name="text">
              <string>Addis Ababa</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gonder</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bahirdar</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jimma</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Adama</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Time of departure:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="timeEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Date of departure: </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Your time of arrival will be</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="arrival_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Type of class </string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="classD_comboBox">
            <item>
             <property name="text">
              <string>Economy class</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Business class</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>First class</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Avalilable Airlines </string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="airlinesD_comboBox">
            <item>
             <property name="text">
              <string>Ethiopian Airlines</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="dateEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Luggage you will be carring(KG)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="luggageD_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="bookD_pushButton">
          <property name="text">
           <string>Book a flight</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Please fill your flight information</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Please select your source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="source_I_comboBox">
            <item>
             <property name="text">
              <string>Canada</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tanzania</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>England</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Please select your destination</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="destination_I_comboBox">
            <item>
             <property name="text">
              <string>Canada</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tanzania</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>England</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Time of departure</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="international_timeEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Date of departure:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDateEdit" name="international_dateEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Your arrival time will be</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="arrival_I_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Type of class</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="class_I_comboBox">
            <item>
             <property name="text">
              <string>First class</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Business class</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Economy class</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Available Airlines</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="airlines_I_comboBox">
            <item>
             <property name="text">
              <string>Quatar Airlines</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Ethipian Airlines</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Emirates Airlines</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Luggage you will be carring(KG)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lug_I_lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="book_I_pushButton">
          <property name="text">
           <string>Book a flight</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
